packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js

```diff
@@ -497,6 +497,103 @@ describe('ReactDOMServerPartialHydration', () => {
     expect(span.className).toBe('hi');
   });
 
+  it('regression test: an update to dehydrated content forces a restart to hydrate it first', async () => {
+    // Replicates a bug where React would fail to restart and hydrate when
+    // `disableSchedulerTimeoutBasedOnReactExpirationTime` is true.
+    jest.resetModuleRegistry();
+
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
+    ReactFeatureFlags.enableSuspenseServerRenderer = true;
+    ReactFeatureFlags.enableSuspenseCallback = true;
+    ReactFeatureFlags.enableFlareAPI = true;
```

Comment:
Was this intentional?

Reply:
Probably not necessary, I just copy pasted from the top of the file for consistency: https://github.com/facebook/react/blob/83102b6e51af196dff7bd2c16e9530eca58f4bb0/packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js#L23-L36